runners/console/index.ts

```diff
@@ -247,6 +247,32 @@ export class Console extends Runner {
         return result;
     }
 
+    runDownloadFile(step: Step, command: Command): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;
+
+        let downloadUrl = command.parameters[0];
+        let installDir = path.join(this.getVariable(this.workspaceDirectory), command.parameters[1]);
+        this.createFolder(installDir, false);
```

Comment:
I think we can make the second parameter optional, too. If not parameter is specified the file will be stored into the workspace directory.